senders/pagerduty/send_test.go

```diff
@@ -43,6 +43,7 @@ func TestBuildEvent(t *testing.T) {
 		baseExpected := pagerduty.V2Event{
 			RoutingKey: contact.Value,
 			Action:     "trigger",
+			DedupKey:   "TriggerID",
```

Comment:
I would like to see more complex example of switching states between `trigger` and `resolved` in tests

Reply:
@litleleprikon Sorry but i can't imagine wthat you want:-), can you clarify? `Dedupkey` is simple trigger id, it doesn't add additional logic

Reply:
It is not about the `DeduceKey` it's about whole tests I just found a first change in tests file to add my comment. My point is that we want to see that action set correctly and this lead us to this additional tests: 

- with trigger in `ERROR` state and action set to `trigger`
- with trigger in `OK` state and action set to `resolve`
- with trigger in `NODATA` state
- with trigger in `WARN` state

In two last cases the action will be `trigger` but is it the right state?